pyqt-apps/siriushla/bl_ap_imgproc/main.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget, QGridLayout, QHBoxLayout, \
+    QVBoxLayout, QGroupBox, QLabel, QSizePolicy
+from siriushla.widgets import SiriusLabel, SiriusLedState, \
+    SiriusLineEdit, PyDMLogLabel, PyDMStateButton, \
+    SiriusConnectionSignal, SiriusSpinbox
+from pydm.widgets import PyDMImageView
+from siriuspy.envars import VACA_PREFIX as _VACA_PREFIX
+from .util import PVS, IMG_PVS, LED_PVS, LOG_PV
+
+class BLImgProc(QWidget):
+    """."""
+
+    def __init__(self, hutch, parent=None, prefix=_VACA_PREFIX):
+        """."""
+        super().__init__(parent=parent)
+        self.setObjectName('SIApp')
+        self.prefix = prefix + ('-' if prefix else '')
+        self.beamline = 'CAX'
+        self.hutch = hutch
+        self.basler = 'BASLER01'
```

Comment:
I think the parameters beamline and basler should be input parameters, as well as the hutch parameter